package.json

```diff
@@ -4,10 +4,16 @@
   "version": "0.7.0",
   "type": "module",
   "homepage": "https://eclipse.github.io/editdor/",
+  "license": "EPL-2.0 OR W3C-20150513",
+  "author": "Eclipse ediTDor <editdor-dev@eclipse.org>",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/eclipse/editdor.git"
+  },
   "scripts": {
     "dev": "cross-env REACT_APP_TARGET_URL=http://localhost:8080 vite",
     "build": "vite build",
-    "build-standalone": "cross-env REACT_APP_TARGET_URL= vite build",
+    "build-standalone": "cross-env REACT_APP_TARGET_URL=https://eclipse.github.io/editdor/ vite build",
```

Comment:
The REACT_APP_TARGET_URL is pretty much what the proxy endpoint query parameter is. So I would leave that blank here `REACT_APP_TARGET_URL=` such that the standalone ediTDor defaults to using the local disk to save TDs/TMs to.

Reply:
ok then we can completely remove the `build-standalone` script?